packages/web-app-files/src/components/SideBar/SideBar.vue

```diff
@@ -153,6 +184,10 @@ export default {
     },
     isRootFolder() {
       return !this.highlightedFile?.path
+    },
+
+    highlightedFileThumbnail() {
+      return this.highlightedFile.thumbnail
```

Comment:
We have a `thumbnail` (small resolution) and `preview` (bigger resolution) for files. The thumbnails are shown in the files list. The previews are shown in the right sidebar. So this should be about previews, not thumbnails, since you're targeting the right sidebar.

Reply:
The preview is being loaded in the FileDetails component. In there, it first checks if the file has the thumbnail so that it knows if it should try to load the bigger one or not. I got a bit confused in this but didn't want to bloat the size of this PR even more with refactoring this so I've just made sure the current behaviour is still working.

https://github.com/owncloud/web/blob/8fa0cf0323f59177fa493f4df7dd04de4dc62328/packages/web-app-files/src/components/SideBar/Details/FileDetails.vue#L5

https://github.com/owncloud/web/blob/8fa0cf0323f59177fa493f4df7dd04de4dc62328/packages/web-app-files/src/components/SideBar/Details/FileDetails.vue#L219

Reply:
Argh. We have an option for disabling thumbnails entirely. If that's the case we apparently kill the preview in the sidebar, just because thumbnails in the list are off.
Understandable though that fixing that is out of scope for this PR. 😞